### PR TITLE
Fix manager user

### PR DESCRIPTION
**Pull request by thiennguyen96841996**

### CODE REVIEW

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

File: resources/views/backend/users/edit.blade.php [hunk 1]
```diff
@@ -19,38 +19,17 @@
...
 
                                         <div class="col-lg-12">
-                                            {{Form::text('email', $user->email, ['class' => 'form-control', 'placeholder' => 'Email' ]) }}
-                                        </div>
-                                    </div>
-                                    <div class="form-group">
-                                        {{ Form::label(__('role'), null, ['class' => 'col-lg-2 control-label']) }}
-                                        <div class="col-lg-12">
-                                            {{ Form::select('role', ['employee' => __('employee_1'), 'manager' => __('manager')], null, ['class' => 'form-control']) }}
+                                            {{Form::text('email', $user->email, ['class' => 'form-control', 'placeholder' => 'Email', 'disabled' => 'disabled']) }}
```
Comment (BunnyPi04) [inline resources/views/backend/users/edit.blade.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by thiennguyen96841996** [comment]
ready

**Pull request merged**